### PR TITLE
IfcProtectiveDevice Rail Import BIMQ 2021-07-21T06:49:55.7962379Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcProtectiveDevice

### CODE REVIEW

File: IFC4x3/Properties/c/CurrentRMS_0Y55X3X$b2LR4CS5G0nLXs/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Maximum rms current of an electric-electronic or electromechanical component at specified ambient temperature.
```
Comment (larswik) [inline IFC4x3/Properties/c/CurrentRMS_0Y55X3X$b2LR4CS5G0nLXs/Documentation.md:1]:
```suggestion
Maximum rms (root mean square) current of an electric-electronic or electromechanical component at specified ambient temperature.
```

File: IFC4x3/Properties/n/NumberOfPoles_2lqxx3IET2UhUPt$MJGW_4/DocProperty.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfPoles_2lqxx3IET2UhUPtMJGW_4" Name="NumberOfPoles" UniqueId="afd3bec3-48e7-427a-b799-dff593420f84" PrimaryDataType="IfcInteger" />
```
Comment (larswik) [inline IFC4x3/Properties/n/NumberOfPoles_2lqxx3IET2UhUPt$MJGW_4/DocProperty.xml:2]:
```suggestion
<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfPoles_2lqxx3IET2UhUPtMJGW_4" Name="NumberOfPoles" UniqueId="afd3bec3-48e7-427a-b799-dff593420f84" PrimaryDataType="IfcCountMeasure" />
```

**Review by larswik — COMMENTED** [review]
I am no expert on the individual properties but added a few suggestions nonetheless.

**Review by larswik — APPROVED** [review]

File: IFC4x3/Properties/n/NumberOfPoles_2lqxx3IET2UhUPt$MJGW_4/DocProperty.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfPoles_2lqxx3IET2UhUPtMJGW_4" Name="NumberOfPoles" UniqueId="afd3bec3-48e7-427a-b799-dff593420f84" PrimaryDataType="IfcCountMeasure" />
+
```
Comment (larswik) [inline IFC4x3/Properties/n/NumberOfPoles_2lqxx3IET2UhUPt$MJGW_4/DocProperty.xml:3]: Should this really be IfcCountMeasure? Just checking (i had some comments about this before, but I think that it was decided to use IfcInteger).

**Review by larswik — APPROVED** [review]

File: IFC4x3/Properties/b/BreakdownVoltageTolerance_36oHQCC0b0jubTadGmGjOS/Documentation.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Nominal value of the tolerance of the dc breakdown voltage of a spark gap.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/b/BreakdownVoltageTolerance_36oHQCC0b0jubTadGmGjOS/Documentation.md]:
```suggestion
Nominal tolerance value of the spark gap dc breakdown voltage.
```

File: IFC4x3/Properties/b/BreakingCapacity_0SRRZKplf9oAKBz3w0a2po/DocProperty.xml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="BreakingCapacity_0SRRZKplf9oAKBz3w0a2po" Name="BreakingCapacity" UniqueId="1c6db8d4-cefa-49c8-a50b-f43e80902cf2" PrimaryDataType="IfcReal" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/b/BreakingCapacity_0SRRZKplf9oAKBz3w0a2po/DocProperty.xml]: Is the PrimaryDataType okay here?

File: IFC4x3/Properties/c/Capacitance_05ATmeM_P3vvDHn98$$8$7/Documentation.md [hunk 6]
```diff
@@ -0,0 +1 @@
+Maximum value of the capacitance between the electrodes of a spark gap at specified frequency and temperature.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/c/Capacitance_05ATmeM_P3vvDHn98$$8$7/Documentation.md]: How about generalizing a bit:
```suggestion
Maximum value of the capacitance between the electrodes at specified frequency and temperature.
```

File: IFC4x3/Properties/c/CurrentRMS_0Y55X3X$b2LR4CS5G0nLXs/DocProperty.xml [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CurrentRMS_0Y55X3Xb2LR4CS5G0nLXs" Name="CurrentRMS" UniqueId="22145843-87f9-4255-b10c-705400c55876" PrimaryDataType="IfcElectricCurrentMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/c/CurrentRMS_0Y55X3X$b2LR4CS5G0nLXs/DocProperty.xml:2]: Do we not have some sort of a similar property available?

File: IFC4x3/Properties/g/GroundingType_0GRwj6SjTAtwY3611gVZUL/Documentation.md [hunk 8]
```diff
@@ -0,0 +1 @@
+The type of ground connection.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/g/GroundingType_0GRwj6SjTAtwY3611gVZUL/Documentation.md]: Is this a typo?
```suggestion
The type of grounding connection.
```

File: IFC4x3/Properties/g/GroundingType_0GRwj6SjTAtwY3611gVZUL/DocProperty.xml [hunk 9]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="GroundingType_0GRwj6SjTAtwY3611gVZUL" Name="GroundingType" UniqueId="106fab46-72d7-4adf-a883-18106a7e3795" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/g/GroundingType_0GRwj6SjTAtwY3611gVZUL/DocProperty.xml]: Penum seems to be missing.

File: IFC4x3/Properties/i/InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ/Documentation.md [hunk 10]
```diff
@@ -0,0 +1 @@
+Minimum resistance between one terminal or several terminals connected together and the case or enclosure of a component at specified voltage
```
Comment (SergejMuhic) [inline IFC4x3/Properties/i/InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ/Documentation.md]:
```suggestion
Minimum resistance between one terminal or several terminals connected together and the case or enclosure of a component at specified voltage.
```

File: IFC4x3/Properties/i/InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ/DocProperty.xml [hunk 11]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ" Name="InsulationResistance" UniqueId="63bd0ed8-ed96-4fc3-ac02-aa99144f569a" PrimaryDataType="IfcElectricResistanceMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/i/InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ/DocProperty.xml]: Could we generalize this and `Pset_SolidStratumCapacity.Resistivity`?

File: IFC4x3/Properties/n/NominalCurrent_1Sia_y4DvB595bUBLM1Dsb/DocProperty.xml [hunk 12]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NominalCurrent_1Sia_y4DvB595bUBLM1Dsb" Name="NominalCurrent" UniqueId="5cb24fbc-10de-4b14-9165-78b55604dda5" PrimaryDataType="IfcElectricCurrentMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NominalCurrent_1Sia_y4DvB595bUBLM1Dsb/DocProperty.xml]: Reuse?

File: IFC4x3/Properties/n/NumberOfPoles_2lqxx3IET2UhUPt$MJGW_4/DocProperty.xml (hunk 3 above)
Comment (SergejMuhic) [reply IFC4x3/Properties/n/NumberOfPoles_2lqxx3IET2UhUPt$MJGW_4/DocProperty.xml:3]: Please look into this comment. Can I resolve?

File: IFC4x3/Properties/n/NumberOfPoles_2lqxx3IET2UhUPt$MJGW_4/Documentation.md [hunk 13]
```diff
@@ -0,0 +1 @@
+Number of poles that the equipment would affect
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NumberOfPoles_2lqxx3IET2UhUPt$MJGW_4/Documentation.md]:
```suggestion
Number of poles that the equipment would affect.
```

File: IFC4x3/Properties/r/RatedFrequency_0xk7ZvULjDVACUV9Qs5Q4T/DocProperty.xml [hunk 14]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RatedFrequency_0xk7ZvULjDVACUV9Qs5Q4T" Name="RatedFrequency" UniqueId="3bb878f9-795b-4d7c-a31e-7c96b615a11d" PrimaryDataType="IfcFrequencyMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/r/RatedFrequency_0xk7ZvULjDVACUV9Qs5Q4T/DocProperty.xml]: Possible reuse of `NominalFrequency`?

File: IFC4x3/Properties/r/RatedFrequency_0xk7ZvULjDVACUV9Qs5Q4T/Documentation.md [hunk 15]
```diff
@@ -0,0 +1 @@
+Frequency of the AC electric power system that the Ripple Control supplied reaches its optimum operating condition
```
Comment (SergejMuhic) [inline IFC4x3/Properties/r/RatedFrequency_0xk7ZvULjDVACUV9Qs5Q4T/Documentation.md]: Something is bothering me with this description, It seems of. Why is "Ripple Control" capitalized?
```suggestion
Frequency of the AC electric power system that the Ripple Control supplied reaches its optimum operating condition.
```

File: IFC4x3/Properties/r/RatedVoltage_0KkDpffYz8L86gzG31TCSi/DocProperty.xml [hunk 16]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RatedVoltage_0KkDpffYz8L86gzG31TCSi" Name="RatedVoltage" UniqueId="14b8dce9-a62f-4854-81aa-f500c174c72c" PrimaryDataType="IfcElectricVoltageMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/r/RatedVoltage_0KkDpffYz8L86gzG31TCSi/DocProperty.xml]: Reuse?

File: IFC4x3/Properties/t/TransformationRatio_0z_ZR$vVrEivWXuanAlBNq/DocProperty.xml [hunk 17]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TransformationRatio_0z_ZRvVrEivWXuanAlBNq" Name="TransformationRatio" UniqueId="3dfa36ff-e5fd-4eb3-9821-e24c4abcb5f4" PrimaryDataType="IfcReal" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/TransformationRatio_0z_ZR$vVrEivWXuanAlBNq/DocProperty.xml]: Would PrimaryDataType be better suited to be one of the `Ifc...RatioMeasure` types?

File: IFC4x3/Properties/t/TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2/DocProperty.xml [hunk 18]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2" Name="TypeOfArcExtinction" UniqueId="6c51b375-bc9b-4649-ae3c-f764aec5f982" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2/DocProperty.xml]: I am missing a Penum.

File: IFC4x3/Properties/t/TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2/Documentation.md [hunk 19]
```diff
@@ -0,0 +1 @@
+Type of arc extinction used in EmergencySwitchOff
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2/Documentation.md]:
```suggestion
Type of arc extinction used in EmergencySwitchOff.
```

File: IFC4x3/Properties/t/Type_2yd82GST551f428rzb7p5q/Documentation.md [hunk 20]
```diff
@@ -0,0 +1 @@
+Type of Spark gap
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/Type_2yd82GST551f428rzb7p5q/Documentation.md]:
```suggestion
Type of Spark gap.
```

File: IFC4x3/Properties/t/Type_2yd82GST551f428rzb7p5q/DocProperty.xml [hunk 21]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Type_2yd82GST551f428rzb7p5q" Name="Type" UniqueId="bc9c8090-71d1-4506-9102-235f651f3174" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/Type_2yd82GST551f428rzb7p5q/DocProperty.xml]: Propose renaming to `SparkGapType`.

File: IFC4x3/PropertyEnumerations/s/PEnum_SparkGapType.xml [hunk 22]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_SparkGapType" Name="PEnum_SparkGapType" UniqueId="47da7c63-4369-4b08-83c9-5c12aa1e5333">
```
Comment (SergejMuhic) [inline IFC4x3/PropertyEnumerations/s/PEnum_SparkGapType.xml:2]: Documentation?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ProtectiveDeviceTypeAntiArcingDevice/DocPropertySet.xml [hunk 23]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_ProtectiveDeviceTypeAntiArcingDevice" UniqueId="c8afd89c-b11c-24ac-9053-b0ee60750e1a" ApplicableType="IfcProtectiveDevice/ANTI" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ProtectiveDeviceTypeAntiArcingDevice/DocPropertySet.xml]: Documentation?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ProtectiveDeviceTypeSparkGap/DocPropertySet.xml [hunk 24]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_ProtectiveDeviceTypeSparkGap" UniqueId="ba48d02b-a912-6f6b-978f-72bfe7c3700e" ApplicableType="IfcProtectiveDevice/SPARK,IfcProtectiveDevice/VOLTAGE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ProtectiveDeviceTypeSparkGap/DocPropertySet.xml]: Documentation?

**Review by SergejMuhic — CHANGES_REQUESTED** [review]

File: IFC4x3/Properties/n/NumberOfPoles_2lqxx3IET2UhUPt$MJGW_4/DocProperty.xml (hunk 3 above)
Comment (fhulin) [reply IFC4x3/Properties/n/NumberOfPoles_2lqxx3IET2UhUPt$MJGW_4/DocProperty.xml:3]: Changed to count Measure. Commit : b5e658cad21e4944423ed95d945268c722299c68

File: IFC4x3/Properties/b/BreakingCapacity_0SRRZKplf9oAKBz3w0a2po/DocProperty.xml (hunk 5 above)
Comment (fhulin) [reply IFC4x3/Properties/b/BreakingCapacity_0SRRZKplf9oAKBz3w0a2po/DocProperty.xml]: Change to Capacitance. Commit : b5e658cad21e4944423ed95d945268c722299c68

File: IFC4x3/Properties/c/CurrentRMS_0Y55X3X$b2LR4CS5G0nLXs/DocProperty.xml (hunk 7 above)
Comment (fhulin) [reply IFC4x3/Properties/c/CurrentRMS_0Y55X3X$b2LR4CS5G0nLXs/DocProperty.xml:2]: I didn't find equivalent property.

File: IFC4x3/Properties/g/GroundingType_0GRwj6SjTAtwY3611gVZUL/DocProperty.xml (hunk 9 above)
Comment (fhulin) [reply IFC4x3/Properties/g/GroundingType_0GRwj6SjTAtwY3611gVZUL/DocProperty.xml]: Changed to single value. Commit : b5e658cad21e4944423ed95d945268c722299c68

File: IFC4x3/Properties/i/InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ/DocProperty.xml (hunk 11 above)
Comment (fhulin) [reply IFC4x3/Properties/i/InsulationResistance_1ZlGxOxPPFmwm2gfaKJrQQ/DocProperty.xml]: Agree! Commit : b5e658cad21e4944423ed95d945268c722299c68

File: IFC4x3/Properties/n/NominalCurrent_1Sia_y4DvB595bUBLM1Dsb/DocProperty.xml (hunk 12 above)
Comment (fhulin) [reply IFC4x3/Properties/n/NominalCurrent_1Sia_y4DvB595bUBLM1Dsb/DocProperty.xml]: Done. Commit : b5e658cad21e4944423ed95d945268c722299c68

File: IFC4x3/Properties/r/RatedFrequency_0xk7ZvULjDVACUV9Qs5Q4T/DocProperty.xml (hunk 14 above)
Comment (fhulin) [reply IFC4x3/Properties/r/RatedFrequency_0xk7ZvULjDVACUV9Qs5Q4T/DocProperty.xml]: Agree. Commit : b5e658cad21e4944423ed95d945268c722299c68

File: IFC4x3/Properties/r/RatedVoltage_0KkDpffYz8L86gzG31TCSi/DocProperty.xml (hunk 16 above)
Comment (fhulin) [reply IFC4x3/Properties/r/RatedVoltage_0KkDpffYz8L86gzG31TCSi/DocProperty.xml]: Done. commit : b5e658cad21e4944423ed95d945268c722299c68

File: IFC4x3/Properties/t/TransformationRatio_0z_ZR$vVrEivWXuanAlBNq/DocProperty.xml (hunk 17 above)
Comment (fhulin) [reply IFC4x3/Properties/t/TransformationRatio_0z_ZR$vVrEivWXuanAlBNq/DocProperty.xml]: Agree. Commit : b5e658cad21e4944423ed95d945268c722299c68

File: IFC4x3/Properties/t/TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2/DocProperty.xml (hunk 18 above)
Comment (fhulin) [reply IFC4x3/Properties/t/TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2/DocProperty.xml]: Change to single value. Commit : b5e658cad21e4944423ed95d945268c722299c68

File: IFC4x3/Properties/t/Type_2yd82GST551f428rzb7p5q/DocProperty.xml (hunk 21 above)
Comment (fhulin) [reply IFC4x3/Properties/t/Type_2yd82GST551f428rzb7p5q/DocProperty.xml]: Agree. Commit : b5e658cad21e4944423ed95d945268c722299c68

File: IFC4x3/PropertyEnumerations/s/PEnum_SparkGapType.xml (hunk 22 above)
Comment (fhulin) [reply IFC4x3/PropertyEnumerations/s/PEnum_SparkGapType.xml:2]: Done. commit : b5e658cad21e4944423ed95d945268c722299c68

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ProtectiveDeviceTypeAntiArcingDevice/DocPropertySet.xml (hunk 23 above)
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ProtectiveDeviceTypeAntiArcingDevice/DocPropertySet.xml]: Done. Commit : b5e658cad21e4944423ed95d945268c722299c68

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ProtectiveDeviceTypeSparkGap/DocPropertySet.xml (hunk 24 above)
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ProtectiveDeviceTypeSparkGap/DocPropertySet.xml]: Done. Commit : b5e658cad21e4944423ed95d945268c722299c68

File: IFC4x3/Properties/b/BreakingCapacity_0SRRZKplf9oAKBz3w0a2po/DocProperty.xml [hunk 25]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="BreakingCapacity_0SRRZKplf9oAKBz3w0a2po" Name="BreakingCapacity" UniqueId="1c6db8d4-cefa-49c8-a50b-f43e80902cf2" PrimaryDataType="IfcElectricCapacitanceMeasure" />
+
```
Comment (larswik) [inline IFC4x3/Properties/b/BreakingCapacity_0SRRZKplf9oAKBz3w0a2po/DocProperty.xml]: Just checking: IfcElectricCapacitanceMeasure as a primary datatype (measured in Farad) when documentation speaks about "the current" (measured in Ampere).

File: IFC4x3/Properties/n/NumberOfPoles_2lqxx3IET2UhUPt$MJGW_4/DocProperty.xml (hunk 3 above)
Comment (larswik) [reply IFC4x3/Properties/n/NumberOfPoles_2lqxx3IET2UhUPt$MJGW_4/DocProperty.xml:3]: Are you sure that it shouldn't be IfcInteger?

File: IFC4x3/Properties/t/TransformationRatio_0z_ZR$vVrEivWXuanAlBNq/Documentation.md [hunk 26]
```diff
@@ -0,0 +1 @@
+The ratio of the actual primary current or voltage to the actual secondary current or voltage
```
Comment (larswik) [inline IFC4x3/Properties/t/TransformationRatio_0z_ZR$vVrEivWXuanAlBNq/Documentation.md]:
```suggestion
The ratio of the actual primary current or voltage to the actual secondary current or voltage.
```

Comment (fhulin) [reply IFC4x3/Properties/t/TransformationRatio_0z_ZR$vVrEivWXuanAlBNq/Documentation.md]: 6561eda44111e366d32b85a216a276c1ab34006b

File: IFC4x3/Properties/b/BreakingCapacity_0SRRZKplf9oAKBz3w0a2po/DocProperty.xml (hunk 25 above)
Comment (fhulin) [reply IFC4x3/Properties/b/BreakingCapacity_0SRRZKplf9oAKBz3w0a2po/DocProperty.xml]: Right. Agree. I change to Current. 6561eda44111e366d32b85a216a276c1ab34006b

**Comment by fhulin** [comment]
@larswik  all issues are solved : 6561eda44111e366d32b85a216a276c1ab34006b

File: IFC4x3/Properties/t/TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2/DocProperty.xml (hunk 18 above)
Comment (SergejMuhic) [reply IFC4x3/Properties/t/TypeOfArcExtinction_1iKRDrl9j6IQuyzsIknVc2/DocProperty.xml]: Is the property required?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ProtectiveDeviceTypeSparkGap/Documentation.md [hunk 27]
```diff
@@ -0,0 +1 @@
+Spark gap properties used in energy domain. The property set can be used by the predefined type SPARKGAP and VOLTAGELIMITER  of IfcProtectiveDevice.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ProtectiveDeviceTypeSparkGap/Documentation.md]:
```suggestion
Spark gap properties used in energy domain. The property set can be used by the predefined type SPARKGAP and VOLTAGELIMITER of IfcProtectiveDevice.
```

**Review by SergejMuhic — COMMENTED** [review]
And one unresolved comment here:
https://github.com/bSI-InfraRoom/IFC-Specification/pull/161#discussion_r776271677

File: IFC4x3/Properties/b/BreakdownVoltageTolerance_36oHQCC0b0jubTadGmGjOS/Documentation.md [hunk 28]
```diff
@@ -0,0 +1 @@
+Nominal tolerance value of the spark gap dc breakdown voltage.
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/b/BreakdownVoltageTolerance_36oHQCC0b0jubTadGmGjOS/Documentation.md]:
```suggestion
Nominal tolerance value of the spark gap DC breakdown voltage.
```

File: IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/DocProperty.xml [hunk 29]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PhaseNumber_1Lt0BWQy1AD811dqTjGf_d" Name="PhaseNumber" UniqueId="55dc02e0-6bc0-4a34-8041-9f476d429fa7" PrimaryDataType="IfcIdentifier" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/DocProperty.xml]: Should this be an identifier or an integer value? just curious?

what is an example value?

File: IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/Documentation.md [hunk 30]
```diff
@@ -0,0 +1 @@
+Number of phase that the equipment operates on.
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/Documentation.md]: Not sure which one so...

If individual referenced phase number:
```suggestion
Phase number that the equipment operates on.
```

If specifying the amount of phases:
```suggestion
Number of phases that the equipment operates on.
```

**Review by AlexBrad1eyCT — CHANGES_REQUESTED** [review]
A few Documentation suggestions after that all good

File: IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/DocProperty.xml (hunk 29 above)
Comment (czapplitec) [reply IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/DocProperty.xml]: This is NumberOfPhases, updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/161/commits/5649d39543bb285cfdf6d42b62ca9afcc5b88438

**Review by AlexBrad1eyCT — APPROVED** [review]
All sorted!

File: IFC4x3/PropertyConstants/a/AIRSPARKGAP_29dgjrRsfDIB_4QxG4zL4X.xml [hunk 31]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AIRSPARKGAP_29dgjrRsfDIB_4QxG4zL4X" Name="AIRSPARKGAP" UniqueId="899eab75-6f6a-4d48-bf84-6bb404f55121">
+	<Documentation>Air spark gap</Documentation>
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/a/AIRSPARKGAP_29dgjrRsfDIB_4QxG4zL4X.xml]:
```suggestion
	<Documentation>Air spark gap.</Documentation>
```

File: IFC4x3/PropertyConstants/g/GASFILLEDSPARKGAP_34OVGxAxP1ZOQjGi8v2Qny.xml [hunk 32]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="GASFILLEDSPARKGAP_34OVGxAxP1ZOQjGi8v2Qny" Name="GASFILLEDSPARKGAP" UniqueId="c461f43b-2bb6-418d-86ad-42c23909ac7c">
+	<Documentation>Gas filled spark gap</Documentation>
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/g/GASFILLEDSPARKGAP_34OVGxAxP1ZOQjGi8v2Qny.xml]:
```suggestion
	<Documentation>Gas filled spark gap.</Documentation>
```

**Review by SergejMuhic — CHANGES_REQUESTED** [review]
Punctuation.

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**